docs/data/sql_functions.yml

```diff
@@ -377,6 +377,12 @@ string:
   - sql: SUBSTR(string, integer1[, integer2])
     table: STRING.substr(INTEGER1[, INTEGER2])
     description: Returns a substring of string starting from position integer1 with length integer2 (to the end by default).
+  - sql: JSON_QUOTE(string)
+    table: STRING.JsonQuote()
+    description: Quotes a string as a JSON value by wrapping it with double quote characters, escaping interior quote and other characters, and returning the result as a utf8mb4 string. If the argument is NULL, the function returns NULL.
+  - sql: JSON_UNQUOTE(string)
+    table: STRING.JsonUnquote()
+    description: Unquotes JSON value and returns the result as a utf8mb4 string. If the argument is NULL, returns NULL. If the value starts and ends with double quotes but is not a valid JSON string literal, an error occurs.
```

Comment:
I didn't get it
I tried ut locally with various of combination and the behaviour doesn't match the description
e.g.
```sql
select JSON_UNQUOTE('"{1,  3]"');
```
returns 
```
{1,  3]
```
did I miss anything?

Reply:
Agreed, I borrowed the definition from MySQL. Perhaps we need a json validity check